data_steward/utils/pipeline_logging.py

```diff
@@ -9,9 +9,57 @@
 
 # Python imports
 import logging
+import logging.config
 import os
+import sys
 from datetime import datetime
 
+# Project imports
+import resources
+
+DEFAULT_LOG_DIR = os.path.join(resources.base_path, 'logs')
+"""Default location for log files"""
+DEFAULT_LOG_LEVEL = logging.INFO
+
+LOG_FORMAT = '%(asctime)s - %(levelname)s - %(name)s - %(message)s'
+LOG_DATEFMT = '%Y-%m-%d %H:%M:%S'
+FILENAME_FMT = '%Y%m%d'
+
+_FILE_HANDLER = 'curation_file_handler'
+"""Identifies the file log handler"""
+_CONSOLE_HANDLER = 'curation_console_handler'
+"""Identifies the console handler"""
+
+_DEFAULT_CONFIG = {
+    'version': 1,
+    'formatters': {
+        'default': {
+            'class': 'logging.Formatter',
+            'format': LOG_FORMAT,
+            'datefmt': LOG_DATEFMT
+        }
+    },
+    'handlers': {
+        _CONSOLE_HANDLER: {
+            'class': 'logging.StreamHandler',
+            'formatter': 'default',
+        },
+        _FILE_HANDLER: {
+            'class': 'logging.FileHandler',
+            'mode': 'a',
+            'formatter': 'default'
+        }
+    },
+    'root': {
+        'handlers': [_CONSOLE_HANDLER, _FILE_HANDLER]
```

Comment:
@mark-velez Actually, I do have another comment.  This is adding a Stream Handler, no matter what.  Stream handlers are fine / great / wanted when a user invokes the code from the command line.  They are not wanted when the script is called by a cron job or some other automated action.  Adding the stream handler should only be done if it is specifically invoked from the command line.  That's why a flag is passed from the command line to the logging setup in the clean cdr engine module.

Reply:
Thanks @lrwb-aou , I added `add_console_handler` to `configure()` which addresses this.